proposals/lip-0030.md

```diff
@@ -325,17 +331,17 @@ accountData = {
     },
     "votes": [
       {
-        "delegateAddress": "c87ea0fa872534cd8fad11193960aa0e5579762af9c2bd23402b6d70fc62f297",
+        "delegateAddress": "83298e0a8a473bb8fd4df30b9230c226c6abdc90fb0030a8439c1d7a44499477",
```

Comment:
Why this and the next 2 delegate addresses are 32 bytes long?

Reply:
Nice spot. I was considering public keys rather than addresses. Will fix